bigquery/dataset.go

```diff
@@ -34,6 +34,19 @@ type Dataset struct {
 	c         *Client
 }
 
+// Collation describes the collation type of a given table or dataset
+type Collation string
+
+// Constants describing available collations
+var (
```

Comment:
The main thing I'm worried about here is whether we should be striving to present an enum-like thing, or if we should treat this more like a string.  Will we end up with a bunch of locale codes and suffixes for behavior?

Reply:
that's a good point. I ended up guiding myself up just from the proto files. Reading the docs around the collation feature it make sense to be more open as a string.